frameworks/kafka/universe/marathon.json.mustache

```diff
@@ -59,6 +59,8 @@
     "CONFIG_TEMPLATE_PATH": "kafka-scheduler",
     "KAFKA_VERSION_PATH": "kafka_2.11-0.10.2.1",
 
+    "KAFKA_ZOOKEEPER_URI": "{{kafka.kafka_zookeeper_uri}}",
```

Comment:
Shouldn't this be `"TASKCFG_ALL_KAFKA_ZOOKEEPER_URI": "{{kafka.kafka_zookeeper_uri}}",` since it needs to be rendered into every Kafka broker's `server.properties`?